src/thunderbots/software/ai/navigator/path_planning_navigator/path_planning_navigator.cpp

```diff
@@ -202,34 +128,59 @@ std::optional<Obstacle> PathPlanningNavigator::obstacleFromAvoidArea(AvoidArea a
             rectangle =
                 Rectangle(world.field().friendlyDefenseArea().neCorner(),
                           Point(-10, world.field().friendlyDefenseArea().seCorner().y()));
-            rectangle.expand(OBSTACLE_INFLATION_DIST);
+            rectangle.expand(
+                Util::DynamicParameters::Navigator::robot_obstacle_inflation_factor
+                    .value() *
+                ROBOT_MAX_RADIUS_METERS);
             return Obstacle(rectangle);
         case AvoidArea::ENEMY_DEFENSE_AREA:
             // We extend the enemy defense area back by several meters to prevent
             // robots going around the back of the goal
             rectangle =
                 Rectangle(world.field().enemyDefenseArea().nwCorner(),
                           Point(10, world.field().enemyDefenseArea().swCorner().y()));
-            rectangle.expand(OBSTACLE_INFLATION_DIST);
+            rectangle.expand(
+                Util::DynamicParameters::Navigator::robot_obstacle_inflation_factor
+                    .value() *
+                ROBOT_MAX_RADIUS_METERS);
             return Obstacle(rectangle);
         case AvoidArea::INFLATED_ENEMY_DEFENSE_AREA:
             rectangle = world.field().enemyDefenseArea();
-            rectangle.expand(OBSTACLE_INFLATION_DIST + 0.2);
+            rectangle.expand(
+                Util::DynamicParameters::Navigator::robot_obstacle_inflation_factor
+                        .value() *
+                    ROBOT_MAX_RADIUS_METERS +
+                0.3);
             return Obstacle(rectangle);
         case AvoidArea::CENTER_CIRCLE:
             return Obstacle::createCircleObstacle(
-                world.field().centerPoint(), world.field().centreCircleRadius(), 1.2);
+                world.field().centerPoint(), world.field().centreCircleRadius(),
+                Util::DynamicParameters::Navigator::robot_obstacle_inflation_factor
+                    .value());
         case AvoidArea::HALF_METER_AROUND_BALL:
-            return Obstacle::createCircleObstacle(world.ball().position(), 0.5, 1.2);
+            return Obstacle::createCircleObstacle(
+                world.ball().position(), 0.5,
```

Comment:
Ok, could we just add a further comment that this is an arbitrary number? (ie. there is no special meaning to it, we just picked it and it seemed to work)

Reply:
It's not arbitrary...

Reply:
Oh, is it from the rules? If that's the case could we leave a comment indicating as such? (if it's not from the rules, where is it from?)

Reply:
Since our createCircleObstacle takes radius in metres, we represent HALF_METER_AROUND_BALL by passing in 0.5. Please see the updated commit. This code is Outdated

Reply:
Oh wow, sorry, somehow I totally missed this was for the `HALF_METER_AROUND_BALL` AvoidArea. My bad!